### PR TITLE
Add 2021.02 release highlights

### PR DESCRIPTION
**Pull request by mcarrano**
Adds release highlights for 2021.02 release.

### CODE REVIEW

File: release-highlights/release-highlights-2021.02/release-highlights.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+![advanced search input](./img/advanced-search.png)
+
+Adds a new search input variant to support attribute-value searching or filtering use cases by combining a search field with a customizable search form.
+
+### [Updated label styling](https://www.patternfly.org/v4/components/label)
+
+![label styles](./img/labels.png)
+
+Borders were added to filled labels to ensure that work well against either white or gray backgrounds.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.02/release-highlights.md:16]:
```suggestion
Borders were added to filled labels to ensure they work well against either white or gray backgrounds.
```

**Comment by evwilkin** [comment]
@mcarrano just added feedback for one issue that looks like a typo or missing word.  Additionally, `Placeholder text in form select` is the only item without any additional descriptive text - the title itself is pretty self explanatory, just making sure this was intentional.

**Comment by mcarrano** [comment]
@evwilkin Thanks for flagging those errors. I made minor updates and also added the auto-complete search enhancement to the 2021.04 release plan.

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**